src/modules/dashboard/components/ActionsPage/VoteWidget/VoteDetails.tsx

```diff
@@ -108,17 +108,15 @@ const VoteDetails = ({
     <div>
       <div className={styles.item}>
         <div className={styles.label}>
-          <div>
-            <FormattedMessage {...MSG.votingMethodLabel} />
-            <QuestionMarkTooltip
-              tooltipText={MSG.votingMethodTooltip}
-              className={styles.help}
-              tooltipClassName={styles.tooltip}
-              tooltipPopperOptions={{
-                placement: 'right',
-              }}
-            />
-          </div>
+          <FormattedMessage {...MSG.votingMethodLabel} />
+          <QuestionMarkTooltip
+            tooltipText={MSG.votingMethodTooltip}
+            className={styles.help}
+            tooltipClassName={styles.tooltip}
+            tooltipPopperOptions={{
+              placement: 'right',
+            }}
+          />
```

Comment:
Very nice. They render in properly in line now

![Screenshot from 2022-07-13 21-25-49](https://user-images.githubusercontent.com/1193222/178805428-da7e7231-c7cb-477b-9b5c-c0021b37cffd.png)